catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/RemoveAllCommand.java

```diff
@@ -189,7 +189,8 @@ private Object executeRemoveAllFromStore() throws Exception {
             console.flush();
 
             // Break out if there are no more records to delete
-            if (amountDeleted < batchSize || batchSize < 1) {
```

Comment:
I don't feel like this is a magic number, but a logical case. If the batch size is asking for less than one record, there's nothing to do. There are no records to perform operations on in each batch.